src/frontends/pytorch/src/op/addmm.cpp

```diff
@@ -17,14 +17,17 @@ namespace op {
 using namespace ov::op;
 
 OutputVector translate_addmm(const NodeContext& context) {
-    num_inputs_check(context, 5, 5);
+    num_inputs_check(context, 3, 5);
     auto input = context.get_input(0);
     auto m1 = context.get_input(1);
     auto m2 = context.get_input(2);
+    auto mm = context.mark_node(std::make_shared<v0::MatMul>(m1, m2));
+    if (context.get_input_size() == 3){
+        return {context.mark_node(std::make_shared<v1::Add>(input, mm))};
+    }
     auto beta = context.get_input(3);
```

Comment:
That is not ideal, you access inputs 3 and 4 without checking that they exist, better would be to check with `!input_is_none` check

Reply:
Added the check